src/test/java/org/folio/marc4ld/mapper/field776/Bibframe2Marc776IT.java

```diff
@@ -0,0 +1,34 @@
+package org.folio.marc4ld.mapper.field776;
```

Comment:
The issue mentioned in the ticket is not reproducible with Bibframe2Marc245IT since it's using different test graph structure. Lets keep this test class